### DISCUSSION TITLE
Repeat injector results in publications with Smilei

### DISCUSSION THREAD

**Discussion by smallrainbow** [SmileiPIC/Smilei#775] [Q&A]
Dear developers,

May I ask a question about the injector which is widely used in lab plasmas and astrophysics. 

How to make a right setups of the injector,? E.g., used in the publication https://doi.org/10.3847/2041-8213/ac30e0
Here, a test namelist has been attached. But I find many Ex,y,z fluctuations at the boundary and the electron px will become vortex...
Please help check it? Thank you very much!

<img width="479" alt="image" src="https://github.com/user-attachments/assets/a445d35c-202a-4ab8-83f3-1f1707259c34">

[shock.zip](https://github.com/user-attachments/files/17929421/shock.zip)

Best regards,
Zhongwei

**Comment by mccoys** [comment]
Did you try to contact the authors?
